cmd/ndm_daemonset/app/command/commands.go

```diff
@@ -59,6 +60,10 @@ func NewNodeDiskManager() (*cobra.Command, error) {
 		"FeatureGates to be enabled or disabled")
 	_ = goflag.CommandLine.Parse([]string{})
 
+	cmd.PersistentFlags().StringVar(&grpc.Address, "api-service-address",
```

Comment:
This flag should be for the start command. Because just running NDM, the user does require to configure anything,